playground/config/responseConfig.js

```diff
@@ -64,6 +64,22 @@ const windowAuthnLoopback = {
   ]
 };
 
+//Windows authenticator with loopback server failfast
+const windowAuthnLoopbackFailfast = {
```

Comment:
With the failfast mock to cover the case of loopback server -> custom URI -> success (identify as a signal in this example), the original loopback server mock should be updated to loopback server -> success (identify). Like:
```
// Windows authenticator with loopback server (make sure to start authenticator mock server from a separate terminal using yarn mock:authenticator)
const windowAuthnLoopback = {
  '/idp/idx/introspect': [
    'identify-with-device-probing-loopback', // 1 (response order)
  ],
  '/idp/idx/authenticators/poll': [
    'identify-with-device-probing-loopback', // 2
    'identify-with-device-probing-loopback', // 3
    'identify', // 4: as a signal of success
  ]
};
```